web/src/Casdoor/Casdoor.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export class Casdoor {
+  constructor(endpoint, organization, application, clientId, redirectUri) {
+    this.endpoint = endpoint;
+    this.organizationName = organization;
+    this.applicationName = application;
+    this.clientId = clientId;
+    this.redirectUri = redirectUri;
+
+    fetch(
```

Comment:
Asynchronous initialization may be better.

Reply:
I think `fetch` is already an async function.

Reply:
All right, it's my wrong.